packages/radix-ui-themes/src/components/base-tab-list.css

```diff
@@ -0,0 +1,140 @@
+.rt-BaseTabList {
+  display: flex;
+  overflow-x: auto;
+  white-space: nowrap;
+
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.rt-BaseTabListLink {
```

Comment:
Super nit: "link" in the class feels a bit specific, should the base call this an item or a trigger instead?